packages/payload/src/fields/setDefaultBeforeDuplicate.ts

```diff
@@ -2,12 +2,14 @@
 // default beforeDuplicate hook for required and unique fields
 import { type FieldAffectingData, type FieldHook, fieldShouldBeLocalized } from './config/types.js'
 
-const unique: FieldHook = ({ value }) => (typeof value === 'string' ? `${value} - Copy` : undefined)
+const hasValue = (value) => typeof value === 'string' && value.trim() !== ''
+
+const unique: FieldHook = ({ value }) => (hasValue(value) ? `${value} - Copy` : undefined)
 const localizedUnique: FieldHook = ({ req, value }) =>
-  value ? `${value} - ${req?.t('general:copy') ?? 'Copy'}` : undefined
-const uniqueRequired: FieldHook = ({ value }) => `${value} - Copy`
+  hasValue(value) ? `${value} - ${req?.t('general:copy') ?? 'Copy'}` : undefined
+const uniqueRequired: FieldHook = ({ value }) => (hasValue(value) ? `${value} - Copy` : undefined)
 const localizedUniqueRequired: FieldHook = ({ req, value }) =>
-  `${value} - ${req?.t('general:copy') ?? 'Copy'}`
+  hasValue(value) ? `${value} - ${req?.t('general:copy') ?? 'Copy'}` : undefined
```

Comment:
Aren't `localizedUniqueRequired` and `localizedUnique` always the same value? I know it is unrelated, but it seems odd. We should clean that up.

On another note, I do wonder what happens when you have a number field that you set to unique on duplicate. Should we set that to undefined if it is marked as unique?